src/Debugger.ts

```diff
@@ -26,14 +26,16 @@ export class Debugger {
     }
   }
 
-  public traceEvent(direction: EventDirection, type: EventType, payload: any, hasHandler: boolean) {
+  public traceEvent(direction: EventDirection, type: EventType, payload: any, destination: any, hasHandler: boolean) {
```

Comment:
for me it looks like `destination` actually is `route` or something like that. We need to find better naming here

Reply:
Routing is definitely a better name yes 👍, thanks for the suggestion will change accordingly.